### DISCUSSION TITLE
No es mucho pero es trabajo honesto 😪

### DISCUSSION THREAD

**Discussion by axlgoze** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#362] [Show and tell]
Hola !! pues nada aquí tratando de completar esta misión que tanto me esta costando, espero puedan checarla y hacerme un poco de feedback, se los agradeceria mucho. 🙏

Se qué no esta igual a la imagen y tambien me faltarón muchas cosas (como los triangulos de fondo) y dudas me sobran como por ejemplo ¿En que casos es mejor usar GRID o FLEX? ¿Es que usar uno u otro es mejor o de "buenas prácticas".... en fin  ¿Algún consejo para ser más rápido programando y ser mejor día a día? con estas prácticas uno verdaderamente aprende y no se si sirva de algo pero dejo enlace al código esperando aportar un granito de arena para todxs aquellos qué también empiezan. 🌱

[**Repositorio**](https://github.com/axlgoze/Front_end_mission_CSS)

[**Página**](https://axlgoze.github.io/Front_end_mission_CSS/)

Por cierto No es responsive, la vdd no tenga idea de como hacerla responsive ¿Algún recurso como un curso un artículo o algo para aprender a hacer sitios responsive? 🤓

**Comment by OscarIslasH** [comment]
Te quedo muy bien, lo unico que note que quedo raro fue que quedo por debajo una cosa

Segun yo se arregla poniendo `br` unas veces para que se baje si no lo quieres volver a posicionar con otros metodos

![Captura de pantalla 2022-03-12 063612](https://user-images.githubusercontent.com/99118114/158018216-df5739a8-6871-455c-9d88-1d218f4a4f37.png)

**Reply by hackit92** [reply]
La verdad no es la mejor práctica hacerlo mediante br, lo ideal es usar margin y padding para asignar espacios entre elementos

**Comment by khisus19** [comment]
Esta muy bien lograda. Felicidades! Según lo poco que sé, es mejor usar grid para el maquetado o el layout de tu página entera y flex para alinear los items dentro del grid.

**Reply by axlgoze** [reply]
No lo habia pensado, gracias por ese dato ✌

**Comment by Ingrzs** [comment]
Responsive es hacer que los elementos de tu pagina se adapten a tamaño movil
Y bueno yo vi que es facil 😅
Solo necesitas la linea de instrucción
Ejemplo
Linea código @media unscreen (tamaño) {
Aqui modificas modificarías el elemento que quieres a tamaño movil
}

**Comment by OmarVenturaP** [comment]
dejame comentarte @axlgoze que te quedo muy bien la página. la verdad esta muy similar a la original. La recomendación que pudiera brindarte es la siguiente:

- Poner al nav un <b>"position: fixed;" </b>para que quede fija en la parte superior aunque bajes en la página y aparte un <b>"opacity: ( );"</b> de menos de 1, puede ser .7 o .8 para que se transparente un poco así como en la original.
-NOTA: Al hacer lo anterior ten en cuenta que tus elementos inferiores subirán un poco. Para ello tienes que darles un <b>"margin-top: ( );"</b> similar a la altura de tu nav.

- También en los puntos de preparations puedes ponerle un <b>"justify-content: space-betwen;"</b> para que tu icono de flecha quede bien distribuido.

Siento que con eso verás una diferencia en primera instancia. pero puedes ir modificándola poco a poco.

**Reply by axlgoze** [reply]
Muchas gracias Omar de echo si tuve muchos problemas con esa flecha de "preparations, le movi de muchas maneras pero al final no conseguí que se distribuyeran bien.
Muy buena lo del fixed no me la sabia 👌 sin duda implementare tus recomendaciones.

**Reply by OmarVenturaP** [reply]
La propiedad de justify space betwen recuerda aplicarsela al div o elemento padre de los iconos. Así te funcionaría
Display flex

Ya a los elementos internos un display flex

**Reply by sergiolopezloya** [reply]
Si usas opacity se hace transparente todo, incluyendo el menú y logo, y no es el caso. Lo ideal es poner un fondo transparente, se hace así:
background-color: rgba(255,255,255,0.7)
Donde 255 x 3 es el color blanco, si cambias esos valores de 0 a 255 obtienes un color diferente rgb (red, green, blue) y el 0.7 es el porcentaje de opacidad, en este caso está transparente al 70%
Si le pones 0% es totalmente transparente.
Espero te sirva ✌️

**Reply by OmarVenturaP** [reply]
Si, de hecho me refería opacidad al elemento padre que tiene fondo blanco. En mi caso no genero problema ya que le aplicó el efecto al bg no a las letras. Pero puedes intentar así como comenta el compañero.

**Comment by Desbarrancadero** [comment]
Te quedó muy bien. Yo sufri con los menus desplegables después de las marcas de vacunas y a ti te quedaron mucho mejores.

**Reply by axlgoze** [reply]
Yo sufri con casi todas las secciones jaja, gracias echale un ojo al código igual te sirve para tu práctica 👌

**Comment by OscarPedroza** [comment]
Un gran trabajo hecho compañero!

**Comment by JuanSalvador73** [comment]
Te está quedando muy bien. Para lo de Responsive, te recomiendo este tutorial:

https://www.youtube.com/watch?v=HtWIrGqbEwI

Falcon Master tiene otros tuts donde explica otros métodos de hacerlo, checa su canal.

**Reply by axlgoze** [reply]
Muchas gracias compañero ✌

**Comment by Ricardo-g-garcia** [comment]
Yo lo de Preparation lo hice con el elemento <details> y <summary> pero no supe como poner la flechita de desplegado a la derecha porque por default vine a la izquierda, aun no subo el repositorio todavia afinando detalles, y preparando campaña de marketing.

**Reply by sergiolopezloya** [reply]
Para las flechas se pueden usar de librerías como Font Awesome, aquí hay unos muy útiles en las 4 direcciones, sólo es cuestión de usar este código:
https://fontawesome.com/search?q=chevron&s=solid

**Reply by axlgoze** [reply]
También puedes descargar iconos gratuitos de páginas como [Flaticon.es](https://www.flaticon.es/) ✌

**Comment by JonathanGrimaldo** [comment]
Excelente trabajo @axlgoze me gusta mucho el uso de los colores, las imágenes y los iconos, se apega mucho al que paso el MC, en lo personal me a encantado tu trabajo! 👏🏻 💯 😄
